src/components/EmojiPicker/EmojiSkinToneList.tsx

```diff
@@ -11,22 +10,23 @@ import CONST from '@src/CONST';
 import EmojiPickerMenuItem from './EmojiPickerMenuItem';
 import getSkinToneEmojiFromIndex from './getSkinToneEmojiFromIndex';
 
+type SkinToneEmoji = {
+    readonly code: string;
+    readonly skinTone: number;
+}
+
 function EmojiSkinToneList() {
     const styles = useThemeStyles();
-    const [highlightedIndex, setHighlightedIndex] = useState(null);
+    const [highlightedIndex, setHighlightedIndex] = useState(-1);
     const [isSkinToneListVisible, setIsSkinToneListVisible] = useState(false);
     const {translate} = useLocalize();
     const [preferredSkinTone, setPreferredSkinTone] = usePreferredEmojiSkinTone();
 
     const toggleIsSkinToneListVisible = useCallback(() => {
         setIsSkinToneListVisible((prev) => !prev);
     }, []);
-
-    /**
-     * Set the preferred skin tone in Onyx and close the skin tone picker
-     * @param {object} skinToneEmoji
-     */
-    function updateSelectedSkinTone(skinToneEmoji) {
+    
+    function updateSelectedSkinTone(skinToneEmoji: SkinToneEmoji) {
```

Comment:
Let's leave descriptions:
```suggestion
/** Set the preferred skin tone in Onyx and close the skin tone picker */
    function updateSelectedSkinTone(skinToneEmoji: SkinToneEmoji) {
```